design-tokens/tokens/component/component.default.json

```diff
@@ -1678,25 +1678,76 @@
         },
         "hover": {
           "boxShadow": {
-            "$type": "shadow",
-            "$value": [
-              {
-                "offsetX": "{base.static.spacing.none}",
-                "offsetY": "{base.static.spacing.none}",
-                "blur": "{base.static.spacing.none}",
-                "spread": "{base.dimension.25}",
-                "color": "{button.secondary.hover.borderColor}",
-                "inset": true
-              }
-            ],
-            "$description": "",
-            "$extensions": {
-              "com.figma": {
-                "hiddenFromPublishing": false,
-                "scopes": [
-                  "ALL_SCOPES"
-                ],
-                "codeSyntax": {}
+            "1": {
```

Comment:
Need to fix this so it condenses the separated shadows back into an array: https://github.com/grommet/hpe-design-system/issues/4155